etc/config/c++.amazon.properties

```diff
@@ -382,21 +382,81 @@ compiler.armv8-full-clang-trunk.options=-target aarch64-linux-gnu --gcc-toolchai
 # allow old URLs to continue to work
 compiler.armv8-full-clang-trunk.alias=armv8.5-clang-trunk
 
-
 # Clang for RISC-V
-group.rvclang.compilers=rv32clang:rv64clang
-group.rvclang.groupName=Clang RISC-V
-group.rvclang.supportsBinary=false
-compiler.rv32clang.exe=/opt/compiler-explorer/clang-trunk/bin/clang++
-compiler.rv32clang.demangler=/opt/compiler-explorer/gcc-snapshot/bin/c++filt
-compiler.rv32clang.objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
-compiler.rv32clang.name=RISC-V rv32gc clang (trunk)
-compiler.rv32clang.options=-target riscv32 -march=rv32gc -mabi=ilp32
-compiler.rv64clang.exe=/opt/compiler-explorer/clang-trunk/bin/clang++
-compiler.rv64clang.demangler=/opt/compiler-explorer/gcc-snapshot/bin/c++filt
-compiler.rv64clang.objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
-compiler.rv64clang.name=RISC-V rv64gc clang (trunk)
-compiler.rv64clang.options=-target riscv64 -march=rv64gc -mabi=lp64
+group.rvcclang.compilers=rv32-clang:rv32-clang1100:rv32-clang1001:rv32-clang1000:rv32-clang-900:rv64-clang:rv64-clang1100:rv64-clang1001:rv64-clang1000:rv64-clang900
```

Comment:
please keep group.rvclang here instead of group.rvcclang

Reply:
Yeah, this was not intended, fixing now.

Reply:
Should be fixed.